### PR TITLE
[Chore] Add Home integration test

### PR DESCRIPTION
**Pull request by sleepylee**
## What happened 👀

Add integration test to the Home screen.
 
## Insight 📝
Test carousel and enter a survey.
 
## Proof Of Work 📹

https://user-images.githubusercontent.com/13435717/134234841-48f147de-6c3a-417d-ac96-d3ff1f254eb5.mov

### CODE REVIEW

**Comment by sleepylee** [comment]
PR is ready for review now, @luongvo @markgravity @manh-t

File: integration_test/home_widget_test.dart [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        // Move back to the 1st survey
+        await tester.fling(pageView, Offset(200, 0), 2000);
+        await tester.pumpAndSettle();
+
+        expect(find.text('Title 1'), findsOneWidget);
+        expect(find.byType(FloatingActionButton), findsOneWidget);
+      }, reportKey: 'scrolling_summary');
+    });
+
+    testWidgets('When choosing a survey, it displays detail', (tester) async {
```
Comment (manh-t) [inline integration_test/home_widget_test.dart:56]: maybe we need more test cases. I.e. tapping on the avatar will show the Drawer, and tapping on Logout button will kick the user back to Login screen 💭

Comment (sleepylee) [reply integration_test/home_widget_test.dart:56]: that's a different scenario I didn't add the side drawer implementation yet 😂

Comment (luongvo) [reply integration_test/home_widget_test.dart:56]: @sleepylee hmm, but you want to close the PR soon 😅

Comment (sleepylee) [reply integration_test/home_widget_test.dart:56]: Screen Shot 2021-09-22 at 14 45 05

@luongvo did I miss anything? 🤷  or did you guys assume all the nice-to-have features == must have?

Comment (sleepylee) [reply integration_test/home_widget_test.dart:56]: https://docs.google.com/document/u/1/d/e/2PACX-1vQERKjvXwKo3GpwRY4Qjc7U0afxjnPRcYEj6VK-scOEiZOYkolirtopUR8cZ2U65akiujAd0LKGp0ZS/pub

**Review by luongvo — APPROVED** [review]
lgtm

File: integration_test/home_widget_test.dart (hunk 1 above)
Comment (manh-t) [reply integration_test/home_widget_test.dart:56]: oh, I see. So this issue is no longer valid. 🐤

**Review by manh-t — APPROVED** [review]

**Pull request merged**